New AC CVE Doc

paolaperaza opened the pull request:
Added a new version of our AC CVE doc. A few notes:

- We originally combined Astronomer Platform + AC CVE's into a single doc, but @petedejoy and I decided to split them up into two
- If this structure works, the aim is to create a parallel CVE doc for the Astronomer Platform
- That I know of, there are no CVE's currently identified for AC 1.10.7 or AC 1.10.10

@ryw Mostly FYI on the format + language in case you have any feedback - will ping you on those Astro Platform CVE's so we can get that doc out there as well.

ryw commented:
should it be ac-cve.md rather than cve-ac.md to align with other ac docs?

paolaperaza commented:
Went back and forth on that but I think it's a good call, thanks @ryw - just fixed slug

paolaperaza commented:
@kaxil Given our convo just now, can you add the 2 CVE's you mentioned for 1.10.5 and beyond to the section of CVE's I just created in this doc, "Apache Airflow Components"?

Admittedly wasn't sure what to name the two sections, so feel free to edit those as well. Currently listed:

1. Apache Airflow Core
2. Apache Airflow Components

cc: @ryw @ashb Feel free to give feedback on the naming of those two sections as well

kaxil commented:
I have added details to this PR (commit: https://github.com/astronomer/docs/pull/189/commits/3d59d32ed1fd0d378aec3e80044237aa0562a8aa) . I think we should remove the Details and just link to external site

paolaperaza commented:
Sweet, thanks @kaxil ! Looks good to me - just added an extra column as well to specify "Component".

@petedejoy Wanna take one last look at this?

paolaperaza commented:
@petedejoy Sweet, thanks for reviewin. In terms of security@astronomer.io emails, Ry set up the account + is getting them right now, but we do need to figure out a smoother process for intaking requests there - I'll post in #security.

ashb commented on v0.13/ac-cve.md:
` 1.10.7-1 to 1.10.7-81.10.6-1 to 1.10.6-21.10.5-1 to 1.10.5-6` Missing some punctuation in that line.

ashb replied:
`1.10.7-1 to 1.10.7-8<br>1.10.6-1 to 1.10.6-2<br>1.10.5-1 to 1.10.5-6` probably?

kaxil replied:
I definitely had `<br>` tags (or atleast I think I had :-D )

kaxil replied:
Updated. Preview:

![image](https://user-images.githubusercontent.com/8811558/80609215-be5bba80-8a2f-11ea-9736-7f122ce39787.png)

ashb commented on v0.13/nav/docs_nav.json:
```diff
@@ -70,6 +70,12 @@
         "slug": "ac-production"
+      },
+      {
+        "title": "Astronomer Certified CVE",
```
What was the thinking for this naming? 

1) It's already under a "Astronomer Certified" heading, so there's a bit of duplication there.
2) I would have expected this to just be "Security"

kaxil replied:
Not sure on this one. @paolaperaza @petedejoy ?

paolaperaza replied:
Hm, yea so I think that's a bit of an issue across our docs to be honest (e.g. we specify "Astronomer Enterprise in a few titles, even though they're under the Enterprise menu).

For now, I chose to name it "Astronomer Certified CVE" because if you're sent a link to that singular doc - it's cleaner to have "Astronomer Certified" specified as the product in question and not just read "Security" (especially if we're gonna have a total of 3 security related docs - Astro platform CVEs (coming soon), "Security Model" (existing) and AC CVE.).

I'd be willing to change it to "Astronomer Certified Security", though if that's clearer than "CVE"? @petedejoy What do you think?

I think once we re-do the website + docs we'll re-visit changing titles more generally, because I do agree it's a bit messy still.

ashb commented on v0.13/ac-cve.md:
```diff
@@ -0,0 +1,42 @@
+|---|---|---|---|---|
+| No CVE's core to Apache Airflow within any Astronomer Certified image identified at this time. |
+
+### Apache Airflow Components
```
Hmmm, in this case these aren't even really components of Airflow, but components of our docker images


 ```suggestion
### Astronomer Certified Docker Images

This section identifies lists security related updates/mitigations in the Astronomer Certified docker images. 
```

And in future we can add another section for Airflow dependencies (for example jQuery).

My thinking for separating them is that non-docker users are responsible for their own updates to these section.

kaxil replied:
Fixed

paolaperaza replied:
@kaxil @ashb Ah ok, yea this makes sense 👍

ashb commented on v0.13/ac-cve.md:
This one only applied to the apline images didn't it?

kaxil replied:
Yes, yes. Good catch

kaxil replied:
Fixed

paolaperaza commented:
Made some changes just now:

- Removed reference to 1.10.6
- Corrected `1.10.7-9` to `1.10.7-10`

@ashb Can you take a quick look so I can merge?
